src/test/java/org/opensearch/neuralsearch/processor/ScoreNormalizationIT.java

```diff
@@ -244,6 +248,95 @@ public void testMinMaxNorm_whenOneShardAndQueryMatches_thenSuccessful() {
         assertHybridSearchResults(searchResponseAsMapGeometricMean, 5, new float[] { 0.6f, 1.0f });
     }
 
+    /**
+     * Using search pipelines with config for z score norm:
+     * {
+     *     "description": "Post processor for hybrid search",
+     *     "phase_results_processors": [
+     *         {
+     *             "normalization-processor": {
+     *                 "normalization": {
+     *                     "technique": "z_score"
+     *                 },
+     *                 "combination": {
+     *                     "technique": "arithmetic_mean"
+     *                 }
+     *             }
+     *         }
+     *     ]
+     * }
+     */
+    @SneakyThrows
+    public void testZScoreNorm_whenOneShardAndQueryMatches_thenSuccessful() {
```

Comment:
Can we add one more test with sorting?

Reply:
Can you create a separate issue for this? Since such test needs to be added for all the techniques